cli/setup.go

```diff
@@ -92,25 +92,21 @@ const (
 	demoRetryPoll         = 30
 	demoUpdatePoll        = 5
 	demoServerCertificate = "/usr/share/doc/mender-client/examples/demo.crt"
-	menderProfessionalURL = "https://hosted.mender.io"
+	hostedMenderURL       = "https://hosted.mender.io"
 
 	// Prompt constants
 	promptWizard = "Mender Client Setup\n" +
 		"===================\n\n" +
 		"Setting up the Mender client daemon: the daemon will " +
 		"regularly poll the server to check for updates and report " +
 		"its inventory data.\nGet started by first configuring the " +
-		"device type and settings for communicating with the server:"
-	promptDone = "Mender is setup successfully. You may now start the " +
-		"daemon by running the `daemon` command."
-	promptDoneRunDaemon = "Starting Mender client…\n" + // NOTE: format
-		"Your device should show up at %s within the next few " +
-		"minutes.\nLog in to your Mender UI at %s to accept it.\n"
-	promptDeviceType         = "Device type (for example raspberrypi3-raspbian): "
-	promptConfirmDev         = "Confirm devicetype as %s? [Y/n] " // NOTE: format
-	promptmenderProfessional = "\nAre you connecting this device to Mender " +
-		"Professional? [Y/n] "
-	promptCredentials = "\nEnter your Mender Professional credentials"
+		"device type and settings for communicating with the server.\n"
+	promptDone         = "Mender setup successfully."
+	promptDeviceType   = "Device type (for example raspberrypi3-raspbian): "
+	promptConfirmDev   = "Confirm devicetype as %s? [Y/n] " // NOTE: format
```

Comment:
As @mirzak and me suggested in the chat, I think we can remove this confirmation step